db/schema.rb

```diff
@@ -293,16 +293,6 @@
     t.index ["user_id"], name: "index_notifications_on_user_id"
   end
 
-  create_table "password_resets", id: :serial, force: :cascade do |t|
```

Comment:
I'm not preserving password resets at all. I think it's OK to delete them here - I don't think we use the history, and any active resets can just be recreated by going to the "Forgot password?" section again.